tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/natives/BuiltinLoadingTest.java

```diff
@@ -31,17 +31,18 @@
     @Test
```

Comment:
Sorry, missed this previously, but should we add the new test to `org.ballerinalang.test.variable.shadowing.ShadowingNegativeTest` instead?

Reply:
sure, will move the test